src/Components/Patient/FileUpload.tsx

```diff
@@ -258,6 +258,11 @@ export const FileUpload = (props: FileUploadProps) => {
     prefetch: !!patientId,
   });
 
+  const { data: consultation } = useQuery(routes.getConsultation, {
+    pathParams: { id: consultationId },
+    prefetch: !!consultationId,
+  });
+
```

Comment:
Can we pass the consultation object as an optional prop to this component instead of making API call just for checking whether to disable or not?

Reply:
There's no consultation object available at that level to pass as prop.
The same thing was done for patient object in the component.